### DISCUSSION TITLE
Program to run Github actions to update my site/page.

### DISCUSSION THREAD

**Discussion by fabiohcnobre** [community/community#49692] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
Community,
Can I schedule it to run the github action every 24 hours and update my site?

My action
https://github.com/inspireterapia/inspireterapia.github.io/blob/9cc60d01c8827492ca40d2e1ce2f03c1a6064802/.github/workflows/jekyll-gh-pages.yml

**Comment by yoannchaudet** [comment]
You can use the `schedule` Actions trigger for doing exactly that: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule.

**Reply by fabiohcnobre** [reply]
My Actions no run 
https://github.com/inspireterapia/inspireterapia.github.io/blob/main/.github/workflows/jekyll-gh-pages.yml

**Reply by yoannchaudet** [reply]
your workflow is invalid, `schedule` needs to be one level higher, not under `push`

**Reply by jge162** [reply]
fix your syntax, indentation syntax error, not position. 

```yaml
on:
  push:
    branches: ["main"]
  schedule:
    - cron: '10 * * * *'
 ```

**Reply by jge162** [reply]
@fabiohcnobre you are good now? looks like it is running with no errors.

**Reply by fabiohcnobre** [reply]
> you are good now? looks like it is running with no errors

but it is not running every hour

**Reply by jge162** [reply]
ah, that has to be a bug with GitHub or cron. based on your syntax it should run every hour 10 min after the hour.

**Reply by fabiohcnobre** [reply]
Yes! Thanks !

**Comment by fabiohcnobre** [comment]
I'm closing because github has been repaired. It's working perfectly thank you all.
